pwiz_tools/Skyline/Model/Koina/Config/KoinaConfig.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0"?>
```

Comment:
The file "KoinaConfig.xml" is not supposed to be committed to the project.
It probably needs to be added to ".gitignore" so that people don't make this mistake.

Reply:
Done: https://github.com/ProteoWizard/pwiz/pull/3004/commits/ec0ca2a0460695003dc8df5329b1ba52f75ab1aa